rdkit/Chem/Draw/IPythonConsole.py

```diff
@@ -427,27 +483,39 @@ def DisableSubstructMatchRendering():
 
 
 def UninstallIPythonRenderer():
-  global _MolsToGridImageSaved, _DrawRDKitBitSaved, _DrawMorganBitSaved, _DrawMorganBitsSaved
   global _rendererInstalled, _methodsToDelete
-  if not _rendererInstalled:
-    return
+  # if not _rendererInstalled:
+  #   return
```

Comment:
Is there is no particular reason to keep these two lines commented out, I would rather delete them.